src/store/slices/articleSlice.ts

```diff
@@ -0,0 +1,48 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Article } from '../../Component/ArticleViewer';
+
+export interface ArticlesState {
+  articles: Article[];
+  currentArticle: Article | null;
+  loading: boolean;
+  error: string | null;
+}
+
+const initialState: ArticlesState = {
+  articles: [],
```

Comment:
add the example articles here, I dont want the any information other what needs to rendered on the front end. in fact as little information the better